pkg/kubelet/dockertools/convert.go

```diff
@@ -74,3 +80,31 @@ func toRuntimeImage(image *docker.APIImages) (*kubecontainer.Image, error) {
 		Size: image.VirtualSize,
 	}, nil
 }
+
+// convert ContainerStatus to api.ContainerStatus.
+func containerStatusToAPIContainerStatus(containerStatus *kubecontainer.ContainerStatus) *api.ContainerStatus {
```

Comment:
@Random-Liu Can we move this to runtime package so it can be reused by rkt as well?

For the DockerPrefix, is there any reason we can't add DockerPrefix in the `kubecontainer.ContainerStatus.Id` ?

cc @yujuhong